Contents/Code/plex_api_helper.py

```diff
@@ -280,15 +274,16 @@ def add_media(item, media_type, media_url_id, media_file=None, media_url=None):
             media_type_dict[media_type]['name'], item.type, item.title, item.ratingKey
         ))
         if media_file:
-            uploaded = upload_media(item=item, method=media_type_dict[media_type]['method'], filepath=media_file)
+            uploaded = upload_media(item=item, method=media_type_dict[media_type]['method'](item), filepath=media_file)
         if media_url:
-            uploaded = upload_media(item=item, method=media_type_dict[media_type]['method'], url=media_url)
+            uploaded = upload_media(item=item, method=media_type_dict[media_type]['method'](item), url=media_url)
     else:
         Log.Warning('No theme songs provided for type: {}, title: {}, rating_key: {}'.format(
             item.type, item.title, item.ratingKey
         ))
 
     if uploaded:
+        settings_hash = general_helper.get_themerr_settings_hash()
```

Comment:
```suggestion
```

Looks like you added the original `settings_hash` back in, so this isn't needed?